com.hack_computer.OperatingSystemApp.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app-id": "com.hack_computer.OperatingSystemApp",
+  "add-extensions": {
+      "com.hack_computer.Clippy.Extension": {
+          "version": "beta",
+          "directory": "clippy",
+          "no-autodownload": false,
+          "autodelete": true
+      }
+  },
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "com.hack_computer.OperatingSystemApp",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--own-name=com.hack_computer.OperatingSystemApp",
+    "--env=GTK3_MODULES=/app/clippy/lib/libclippy-module.so",
+    "--talk-name=com.hack_computer.Clubhouse",
+    "--talk-name=com.hack_computer.GameStateService",
+    "--talk-name=com.hack_computer.HackSoundServer",
+    "--talk-name=org.gnome.Shell",
+    "--talk-name=com.hack_computer.HackableAppsManager",
+    "--talk-name=com.hack_computer.hack"
+  ],
+  "modules": [
+    {
+      "name": "clippy-ext",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir /app/clippy"
+      ],
+      "sources": []
+    },
+    {
+      "name": "toyapp",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dapp-id=com.hack_computer.OperatingSystemApp"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/endlessm/hack-toy-apps.git",
+          "commit": "b416928be0a471f2289d561931d5a93cef4fa7fe",
+          "branch": "master"
```

Comment:
if you're pointing to a commit you shouldn't necessarily need a branch - only a nitpick though

Reply:
okay.